src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts

```diff
@@ -1,31 +1,57 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
```

Comment:
Please break down the tests on this component into smaller describe easier to follow.